set.go

```diff
@@ -26,6 +26,10 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+// SetConcateTypeBits defines concatination bits, originally defined in
```

Comment:
here and elsewhere: fix typo: SetConcatTypeBits (no “e” after Concat)

Reply:
done